dhis-2/dhis-web-api/src/main/resources/openapi/ProgramMessageController.md

```diff
@@ -0,0 +1,61 @@
+# Send Message
+
+## Specific endpoints
+
+### `sendMessages`
+
+Send event or enrollment messages.
+
+### `getScheduledSentMessage`
+
+Get all of those scheduled messages which were sent successfully.
+
+### `getProgramMessages`
+
+Get program messages matching given query criteria.
+
```

Comment:
You can have a look at this specs https://github.com/dhis2/dhis2-core/blob/master/dhis-2/dhis-web-api/src/main/resources/openapi/RelationshipsExportController.md

It would be good to show the parameters that are specific for one endpoint, linked to it and then having a list of common parameters.